GitCommands/Remotes/GitRemoteManager.cs

```diff
@@ -274,8 +274,26 @@ public GitRemoteSaveResult SaveRemote(GitRemote remote, string remoteName, strin
             bool remoteDisabled = false;
             if (creatingNew)
             {
-                output = module.AddRemote(remoteName, remoteUrl);
-                updateRemoteRequired = true;
+                if (GetEnabledRemoteNames().FirstOrDefault(r => r == remoteName) != null)
+                {
+                    return new GitRemoteSaveResult($"An active remote named \"{remoteName}\" already exists.", false);
```

Comment:
Note that we want to return an error result right away, and not fall through below, otherwise the remote will get updated.